Microsoft.HealthVault.Fhir/Microsoft.HealthVault.Fhir.csproj

```diff
@@ -7,7 +7,13 @@
 
   <ItemGroup>
     <PackageReference Include="Hl7.Fhir.STU3" Version="0.93.4" />
-    <PackageReference Include="Microsoft.HealthVault" Version="1.65.20508.1-preview" />
+    <PackageReference Include="NodaTime" Version="2.0.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.HealthVault">
+      <HintPath>..\..\healthvault-dotnetstandard-sdk\Microsoft.HealthVault\bin\Debug\netstandard1.4\Microsoft.HealthVault.dll</HintPath>
+    </Reference>
```

Comment:
I will fix this, the problem is that the official nuget doesn't have some fixes I need. I am working with David to figure out what to do regarding that.